Catch exception for broken taupage yaml

```
ERROR [pool-4-thread-1] o.z.s.f.PluginEventsProcessor - expected '<document start>', but found Scalar
   in 'string', line 1, column 25:
      "#taupage-ami-config\n" +
                              ^
  
  com.google.common.util.concurrent.UncheckedExecutionException: expected '<document start>', but found Scalar
   in 'string', line 1, column 25:
      "#taupage-ami-config\n" +
                              ^
  
  #011at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2207) ~[guava-19.0.jar!/:?]
  #011at com.google.common.cache.LocalCache.get(LocalCache.java:3953) ~[guava-19.0.jar!/:?]
  #011at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3957) ~[guava-19.0.jar!/:?]
  #011at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4875) ~[guava-19.0.jar!/:?]
  #011at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4881) ~[guava-19.0.jar!/:?]
  #011at org.zalando.stups.fullstop.plugin.provider.impl.TaupageYamlProviderImpl.apply(TaupageYamlProviderImpl.java:87) ~[fullstop-instance-plugin-support-1.2.0-SNAPSHOT.jar!/:?]
  #011at org.zalando.stups.fullstop.plugin.provider.impl.TaupageYamlProviderImpl.apply(TaupageYamlProviderImpl.java:27) ~[fullstop-instance-plugin-support-1.2.0-SNAPSHOT.jar!/:?]
  #011at org.zalando.stups.fullstop.plugin.impl.EC2InstanceContextImpl.getTaupageYaml(EC2InstanceContextImpl.java:183) ~[fullstop-instance-plugin-support-1.2.0-SNAPSHOT.jar!/:?]
  #011at org.zalando.stups.fullstop.plugin.impl.EC2InstanceContextImpl.getApplicationId(EC2InstanceContextImpl.java:130) ~[fullstop-instance-plugin-support-1.2.0-SNAPSHOT.jar!/:?]
  #011at org.zalando.stups.fullstop.plugin.LifecyclePlugin.process(LifecyclePlugin.java:54) ~[fullstop-application-lifecycle-plugin-1.2.0-SNAPSHOT.jar!/:?]
  #011at java.util.ArrayList.forEach(ArrayList.java:1249) ~[?:1.8.0_66-internal]
  #011at org.zalando.stups.fullstop.plugin.AbstractEC2InstancePlugin.processEvent(AbstractEC2InstancePlugin.java:36) ~[fullstop-instance-plugin-support-1.2.0-SNAPSHOT.jar!/:?]
  #011at org.zalando.stups.fullstop.PluginEventsProcessor.doProcess(PluginEventsProcessor.java:52) ~[fullstop-processing-1.2.0-SNAPSHOT.jar!/:?]
  #011at org.zalando.stups.fullstop.PluginEventsProcessor.doProcess(PluginEventsProcessor.java:43) ~[fullstop-processing-1.2.0-SNAPSHOT.jar!/:?]
  #011at java.util.ArrayList.forEach(ArrayList.java:1249) [?:1.8.0_66-internal]
  #011at org.zalando.stups.fullstop.PluginEventsProcessor.process(PluginEventsProcessor.java:33) [fullstop-processing-1.2.0-SNAPSHOT.jar!/:?]
  #011at com.amazonaws.services.cloudtrail.processinglibrary.reader.EventReader.emitEvents(EventReader.java:229) [aws-cloudtrail-processing-library-1.0.1-ZAL-2.jar!/:?]
  #011at com.amazonaws.services.cloudtrail.processinglibrary.reader.EventReader.processSource(EventReader.java:156) [aws-cloudtrail-processing-library-1.0.1-ZAL-2.jar!/:?]
  #011at com.amazonaws.services.cloudtrail.processinglibrary.AWSCloudTrailProcessingExecutor$ScheduledJob$1.run(AWSCloudTrailProcessingExecutor.java:178) [aws-cloudtrail-processing-library-1.0.1-ZAL-2.jar!/:?]
  #011at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_66-internal]
  #011at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_66-internal]
  #011at java.lang.Thread.run(Thread.java) [?:1.8.0_66-internal]
  Caused by: org.yaml.snakeyaml.parser.ParserException: expected '<document start>', but found Scalar
```

--- a/fullstop-plugins/fullstop-instance-plugin-support/src/main/java/org/zalando/stups/fullstop/plugin/provider/impl/TaupageYamlProviderImpl.java
+++ b/fullstop-plugins/fullstop-instance-plugin-support/src/main/java/org/zalando/stups/fullstop/plugin/provider/impl/TaupageYamlProviderImpl.java
@@ -11,6 +11,7 @@ import com.google.common.cache.CacheLoader;
 import com.google.common.cache.LoadingCache;
 import org.slf4j.Logger;
 import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.parser.ParserException;
 import org.yaml.snakeyaml.scanner.ScannerException;
 import org.zalando.stups.fullstop.plugin.EC2InstanceContext;
 import org.zalando.stups.fullstop.plugin.provider.TaupageYamlProvider;
@@ -71,7 +72,7 @@ public class TaupageYamlProviderImpl implements TaupageYamlProvider {
             } catch (AmazonClientException e) {
                 log.warn("Could not get Taupage YAML for instance: " + instanceId, e);
                 return empty();
-            } catch (ScannerException s) {
+            } catch (ScannerException | ParserException | IllegalArgumentException s)   {
                 log.warn("Taupage YAML is not valid for instance: " + instanceId, s);
                 return empty();
             }

